doctr/models/classification/vit/pytorch.py

```diff
@@ -29,7 +29,29 @@
 }
 
 
-class VisionTransformer(nn.Module):
+class ClassifierHead(nn.Module):
+    """Classifier head for Vision Transformer
+
+    Args:
+        in_channels: number of input channels
+        num_classes: number of output classes
+    """
+
+    def __init__(
+        self,
+        in_channels: int,
+        num_classes: int,
+    ) -> None:
+        super().__init__()
+
+        self.head = nn.Linear(in_channels, num_classes)
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        # (batch_size, num_classes) cls token
+        return self.head(x[:, 0])
```

Comment:
Mmmmh, what's the difference with

```python
head = nn.Linear(in_channels, num_classes)
...
out = head(x[:, 0])
```

(Linear actually supports higher dimensions than 2, we can reshape it afterwards I think)

Reply:
It would be cleaner to add a squeeze or flatten layer in the sequential, rather than creating a class that is doing 99% the same as a Linear :)